src/gt4py/next/iterator/transforms/pass_manager.py

```diff
@@ -191,18 +127,13 @@ def apply_common_transforms(
             ir = unrolled
             ir = CollapseListGet().visit(ir)
             ir = NormalizeShifts().visit(ir)
-            ir = _inline_lifts(ir, LiftMode.FORCE_INLINE)
+            # this is required as nested neighbor reductions can contain lifts, e.g.,
+            # `neighbors(V2Eₒ, ↑f(...))`
+            ir = inline_lifts.InlineLifts().visit(ir)
             ir = NormalizeShifts().visit(ir)
         else:
             raise RuntimeError("Reduction unrolling failed.")
 
-    ir = EtaReduction().visit(ir)
-    ir = ScanEtaReduction().visit(ir)
```

Comment:
what happened to these transformations?

Reply:
This is one of the advantages of GTIR. They are not needed at all.